dasharo-security/wifi-bluetooth-switch.robot

```diff
@@ -54,6 +56,8 @@ WBS002.001 Wifi and Bluetooth card power switch enabled (Ubuntu 22.04)
     Set Option State    ${security_menu}    Enable Wi-Fi + BT radios    ${TRUE}
     Save Changes And Reset
     Login To Linux
+    Detect Or Install Package    pciutils
```

Comment:
@JanPrusinowski Please inspect the implemenation of this keyword. I am afraid you need to be root, so first you need to use `Switch To Root User` as well.

Reply:
Added where it was required